src/main/java/com/github/vkorobkov/jfixturescmd/utils/ExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.github.vkorobkov.jfixturescmd.utils;
+
+import com.beust.jcommander.ParameterException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class ExceptionHandler {
+    private ExceptionHandler() {
+    }
+
+    public static void handleException(Exception e) {
+        if (e instanceof ParameterException) {
+            log.error("Failed to parse command line arguments: " + e.getMessage());
+        } else {
+            log.error(e.getMessage());
```

Comment:
Let's write a message only for `ParameterException` and for exceptions which core throws: `LoaderException` and `ProcessorException`?